wasm-chain-integration/src/utils.rs

```diff
@@ -250,8 +250,10 @@ pub fn generate_contract_schema_v0(module_bytes: &[u8]) -> ExecResult<schema::Mo
 }
 
 /// Tries to generate schemas for parameters and return values of methods for a
-/// V1 contract.
-pub fn generate_contract_schema_v1(module_bytes: &[u8]) -> ExecResult<schema::ModuleV1> {
+/// versioned module v0.
+pub fn generate_contract_schema_versioned_v0(
```

Comment:
```suggestion
/// versioned module v1.
pub fn generate_contract_schema_versioned_v1(
```
Shouldn't this be named v1?

Reply:
It should be without the version suffix